service-stubs/registry/org.wso2.carbon.registry.resource.stub/src/main/resources/ResourceAdminService.wsdl

```diff
@@ -759,6 +759,20 @@
                     </xs:sequence>
                 </xs:complexType>
             </xs:element>
+            <xs:element name="getProperties">
+                <xs:complexType>
+                    <xs:sequence>
+                        <xs:element minOccurs="0" name="resourcePath" nillable="true" type="xs:string"/>
```

Comment:
Can the resourcePath be nillable?

Reply:
This was done to maintain consistency among other operations.